modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/model/CalendarPortletLayoutTypeAccessPolicy.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.calendar.web.internal.model;
+
+import com.liferay.calendar.constants.CalendarPortletKeys;
+import com.liferay.portal.kernel.model.Layout;
+import com.liferay.portal.kernel.model.LayoutTypeAccessPolicy;
+import com.liferay.portal.kernel.model.Portlet;
+import com.liferay.portal.kernel.model.impl.DefaultLayoutTypeAccessPolicyImpl;
+import com.liferay.portal.kernel.security.permission.PermissionChecker;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Leonardo Barros
```

Comment:
Hey @carolmariaabb, can you change it to your name before sending this PR to Brian?